Source/WTF/wtf/FastMalloc.cpp

```diff
@@ -34,7 +34,8 @@
 #endif
 
 #if OS(WINDOWS)
-#include <windows.h>
+#include <Windows.h>
```

Comment:
Is `Windows.h` correct? I was thinking that it is `windows.h`.

Reply:
Really? I keep believing it is `Windows.h` for decades. Let me check.

Reply:
Actually this part cannot be compiled yet. I remove this part from the change and keep researching it.

Reply:
In WebKit code base, lowercase (260) vs Capitalized (5). Lowercase is more consistent in the codebase.